### PR TITLE
Update changelog based on user feedback

### PR DESCRIPTION
**Pull request by EoinShaughnessy**
Fixes [#2431](https://github.com/alphagov/govuk-frontend/issues/2431).

We recently tested an early version of 4.0 release notes with some users. [See issue for full details.](https://github.com/alphagov/govuk-frontend/issues/2351)

This PR adds content into the changelog, with changes based on their feedback.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -91,6 +95,24 @@ Replace any use of this class with the `govuk-tag--grey` class.
 
 This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
 
+### Optional changes
```
Comment (vanitabarrett) [inline CHANGELOG.md:98]: I think there are a couple of other optional changes we need to add? Listed in this doc https://docs.google.com/document/d/1sANEJKi1PQmjwipdxViIz_gx5PJM_rUTfwelfAU8kcc/edit

Comment (EoinShaughnessy) [reply CHANGELOG.md:98]: Ah sorry, my bad. I was only using this PR for changes based on user feedback on the ['test' notes](https://docs.google.com/document/d/1zXF8tRgbRJm8863-laNLB9ZcB4AbWdkNI-IGDtCth2U/edit?pli=1#). I can add those other ones in too.

Comment (EoinShaughnessy) [reply CHANGELOG.md:96]: @vanitabarrett Just added them now. Atom behaved a bit oddly - I'd edit something and it made the formatting go haywire. Think I've sorted it now though (said in hope).

File: CHANGELOG.md [hunk 2]
```diff
@@ -91,6 +93,51 @@ Replace any use of this class with the `govuk-tag--grey` class.
...
+
+You do not need to do anything if you're using Nunjucks macros.
+
+#### Change date input from `type=”number”` to `inputmode=numeric`
+
+There are some known issues with inputs of `type="number”`. [Read our research into the issues with `type=“number”`.](https://github.com/alphagov/govuk-frontend/issues/1449#issuecomment-503186819)
+
+If you’re not using Nunjucks macros, remove the `type=“number”` attribute from the date input component.
+
+ Replace it with `type=“text”` and `inputmode=“numeric”`, for example:.
```
Comment (vanitabarrett) [inline CHANGELOG.md:108]:
```suggestion
 Replace it with `type="text"` and `inputmode="numeric"`, for example:
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -91,6 +93,51 @@ Replace any use of this class with the `govuk-tag--grey` class.
 
 This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
 
+### Optional changes
+
+We've recently made some other changes to GOV.UK Frontend. While these are not breaking changes, implementing them will make your service work better.
+
+You do not need to do anything if you're using Nunjucks macros.
+
+#### Change date input from `type=”number”` to `inputmode=numeric`
```
Comment (vanitabarrett) [inline CHANGELOG.md:102]:
```suggestion
#### Change date input from `type="number"` to `inputmode=numeric`
```

Comment (vanitabarrett) [reply CHANGELOG.md:102]: Should we update the title to include `type="text"` too? Otherwise the title sounds a misleading

File: CHANGELOG.md [hunk 4]
```diff
@@ -91,6 +93,51 @@ Replace any use of this class with the `govuk-tag--grey` class.
...
+
+There are some known issues with inputs of `type="number”`. [Read our research into the issues with `type=“number”`.](https://github.com/alphagov/govuk-frontend/issues/1449#issuecomment-503186819)
```
Comment (vanitabarrett) [inline CHANGELOG.md:104]:
```suggestion
There are some known issues with inputs of `type="number"`. [Read our research into the issues with `type="number"`.](https://github.com/alphagov/govuk-frontend/issues/1449#issuecomment-503186819)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -91,6 +93,51 @@ Replace any use of this class with the `govuk-tag--grey` class.
...
+
+If you’re not using Nunjucks macros, remove the `type=“number”` attribute from the date input component.
```
Comment (vanitabarrett) [inline CHANGELOG.md:106]:
```suggestion
If you’re not using Nunjucks macros, remove the `type="number"` attribute from the date input component.
```

**Review by vanitabarrett — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (EoinShaughnessy) [reply CHANGELOG.md:102]: Ah yes. Like this?
```suggestion
#### Change date input from `type=”number”` to `type="text"` and `inputmode=numeric`
```

Comment (vanitabarrett) [reply CHANGELOG.md:102]: Yep, although we should probably also be consistent with the quotes, e.g: `type="text"` and `inputmode="numeric"`

Comment (EoinShaughnessy) [reply CHANGELOG.md:102]: Definitely! Thanks.

```suggestion
#### Change date input from `type=”number”` to `inputmode="numeric"`
```

**Comment by EoinShaughnessy** [comment]
@vanitabarrett Thanks for reviewing! Have actioned comments.

File: CHANGELOG.md [hunk 6]
```diff
@@ -41,9 +41,11 @@ This change was introduced in [pull request #2385: Remove deprecated `govuk-main
 
 #### Remove deprecated `$govuk-border-width-form-element-error` variable
```
Comment (vanitabarrett) [inline CHANGELOG.md:42]: I don't know if I'm getting confused, but I think there was some additional text here to explain why the change was made? I  think it was included in the first version of this PR, so potentially might have got lost when the newer changes were added? Or perhaps I'm imagining them completely 😅

File: CHANGELOG.md [hunk 7]
```diff
@@ -91,6 +93,51 @@ Replace any use of this class with the `govuk-tag--grey` class.
 
 This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
 
+### Optional changes
+
+We've recently made some other changes to GOV.UK Frontend. While these are not breaking changes, implementing them will make your service work better.
+
+You do not need to do anything if you're using Nunjucks macros.
+
+#### Change date input from `type=”number”` to `inputmode="numeric"`
```
Comment (vanitabarrett) [inline CHANGELOG.md:102]:
```suggestion
#### Change date input from `type="number"` to `inputmode="numeric"`
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -91,6 +93,51 @@ Replace any use of this class with the `govuk-tag--grey` class.
...
+
+```
+
+This was added in [pull request #1704: Update date input to use `type=text` and  `inputmode=numeric`](https://github.com/alphagov/govuk-frontend/pull/1704).
+
+#### Fix fallback logo so Chrome will not flag it to screen readers
+
+If you’re not using Nunjucks macros, you will need to change the SVG markup within the header, footer and button components. We’ve added this fix so screen readers will not announce the fallback image to users.
+
+Replace `role="presentation"` with aria-hidden="true". For example:
```
Comment (vanitabarrett) [inline CHANGELOG.md:121]:
```suggestion
Replace `role="presentation"` with `aria-hidden="true"`. For example:
```

File: CHANGELOG.md (hunk 6 above)
Comment (EoinShaughnessy) [reply CHANGELOG.md:42]: Haha, no, you're totally right - it's in the [release notes we tested with users](https://docs.google.com/document/d/1zXF8tRgbRJm8863-laNLB9ZcB4AbWdkNI-IGDtCth2U/edit?pli=1#heading=h.icv1a7bx25ly). Sorry, I might have lost it somewhere along the way. Will add back in.

**Comment by EoinShaughnessy** [comment]
@vanitabarrett Thanks for reviewing! Have added in your edits to replace curly quotes, plus the content explaining our changes around the `$govuk-border-width-form-element-error` variable.

**Review by vanitabarrett — APPROVED** [review]
